### ISSUE TITLE
Update FAQ with VERY basic info such as how to send and receive Decred

### ISSUE THREAD

**Issue by gratefulcheddar** [decred/dcrdocs#39]
Update FAQ with VERY basic info such as how to send and receive Decred

**Comment by animedow** [comment]
I would like to work on this.
Also, based on the kind of questions we get in the slack channel I think it's important to include what the PoS system is in layman's terms.

**Comment by jcvernaleo** [comment]
@animedow Great, go for it.  I'll assign this issue to you.

**Comment by jcvernaleo** [comment]
Well, actually, it looks like I can't assign an issue to someone who isn't already a contributer (or member).  But we can see from the comments that you plan on working on this one.
